src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+  const declarations = styles.split(';');
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':');
```

Comment:
When splitting the declaration by ':', there might be cases where the value itself contains colons (e.g., URLs). Consider using a more robust parsing method to handle such cases.